pkgs/by-name/li/linien-gui/package.nix

```diff
@@ -4,16 +4,44 @@
   qt5,
 }:
 
-python3.pkgs.buildPythonApplication rec {
+let
+  loosenNumpy =
+    p: pattern:
+    p.overridePythonAttrs (old: {
+      postPatch =
+        (old.postPatch or "")
+        # Extra empty line is here due to potential last line of existing
+        # postPatch ending with `\`.
```

Comment:
Where is that the case? Should be fixed.

Reply:
> Where is that the case? Should be fixed.

In `pandas`. I was thinking about this at first, but then I realized it would cause > 1000 rebuilds...